backend/song/test/test_models.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+pytestmark = pytest.mark.django_db
+
+class TestModel:
+    def test_check_category_creation(self, category):
+
+        assert category.category_name == "rumba"
```

Comment:
also, you can check if the model is category is an instance of the Category model